boot/bootutil/include/bootutil/caps.h

```diff
@@ -35,22 +35,23 @@ uint32_t bootutil_get_caps(void);
 
 #define BOOTUTIL_CAP_RSA2048                (1<<0)
 #define BOOTUTIL_CAP_ECDSA_P256             (1<<1)
-#define BOOTUTIL_CAP_SWAP_USING_SCRATCH     (1<<2)
-#define BOOTUTIL_CAP_OVERWRITE_UPGRADE      (1<<3)
-#define BOOTUTIL_CAP_ENC_RSA                (1<<4)
-#define BOOTUTIL_CAP_ENC_KW                 (1<<5)
-#define BOOTUTIL_CAP_VALIDATE_PRIMARY_SLOT  (1<<6)
-#define BOOTUTIL_CAP_RSA3072                (1<<7)
-#define BOOTUTIL_CAP_ED25519                (1<<8)
-#define BOOTUTIL_CAP_ENC_EC256              (1<<9)
-#define BOOTUTIL_CAP_SWAP_USING_MOVE        (1<<10)
-#define BOOTUTIL_CAP_DOWNGRADE_PREVENTION   (1<<11)
-#define BOOTUTIL_CAP_ENC_X25519             (1<<12)
-#define BOOTUTIL_CAP_BOOTSTRAP              (1<<13)
-#define BOOTUTIL_CAP_AES256                 (1<<14)
-#define BOOTUTIL_CAP_RAM_LOAD               (1<<15)
-#define BOOTUTIL_CAP_DIRECT_XIP             (1<<16)
-#define BOOTUTIL_CAP_HW_ROLLBACK_PROT       (1<<17)
+#define BOOTUTIL_CAP_ECDSA_P384             (1<<2)
+#define BOOTUTIL_CAP_SWAP_USING_SCRATCH     (1<<3)
+#define BOOTUTIL_CAP_OVERWRITE_UPGRADE      (1<<4)
+#define BOOTUTIL_CAP_ENC_RSA                (1<<5)
+#define BOOTUTIL_CAP_ENC_KW                 (1<<6)
+#define BOOTUTIL_CAP_VALIDATE_PRIMARY_SLOT  (1<<7)
+#define BOOTUTIL_CAP_RSA3072                (1<<8)
+#define BOOTUTIL_CAP_ED25519                (1<<9)
+#define BOOTUTIL_CAP_ENC_EC256              (1<<10)
+#define BOOTUTIL_CAP_SWAP_USING_MOVE        (1<<11)
+#define BOOTUTIL_CAP_DOWNGRADE_PREVENTION   (1<<12)
+#define BOOTUTIL_CAP_ENC_X25519             (1<<13)
+#define BOOTUTIL_CAP_BOOTSTRAP              (1<<14)
+#define BOOTUTIL_CAP_AES256                 (1<<15)
+#define BOOTUTIL_CAP_RAM_LOAD               (1<<16)
+#define BOOTUTIL_CAP_DIRECT_XIP             (1<<17)
+#define BOOTUTIL_CAP_HW_ROLLBACK_PROT       (1<<18)
```

Comment:
This breaks compatibility with everything

Reply:
This could be used by downstream users to communicate capabilities with applications or as part of regression tests. It is common sense that once bitmask values are set, they are not changed or reordered, and new values are appended to the end of the list, not pushed into the middle of the list causing 16 other entries to be shifted around.

Reply:
If used correctly, it should not cause any problems. But given its importance, the patch can be easily updated to append the new cap to the end instead. I believe the point here was to improve readability.
@Roolli may I ask you to update the patch accordingly?  Please also revert the modifications of 206b914 on this file.

Reply:
I don't see any particular reason to change this. True that it would need to be compiled. Please just add the new one at the end.